### PR TITLE
draft updates to some of the config docs

### PR DESCRIPTION
**Pull request by jantley-ibm**

### CODE REVIEW

File: ref/general/configuration/console-oauth.adoc [hunk 1]
```diff
@@ -3,31 +3,31 @@
 :page-title: Configuring Kabanero Console with OAuth
 :linkattrs:
 :sectanchors:
-= Configuring the Kabanero Console with OAuth
+= Configuring the product console with OAuth
 
-After Kabanero is installed, you can enable optional collection management features via the Kabanero Console by configuring GitHub OAuth. This setup will allow the console to verify users via GitHub OAuth and communicate with the collections server.
+You can enable optional stack hub management features via the product console by configuring GitHub OAuth. This setup will allow the console to verify users via GitHub OAuth and communicate with the stack hub server.
```
Comment (stephenkinder) [inline ref/general/configuration/console-oauth.adoc]: "stack hub server" is odd language.  @alohr51 what are we trying to say here?  What server are we talking about?

File: ref/general/configuration/github-authorization.adoc [hunk 2]
```diff
@@ -6,15 +6,15 @@
...
 If you do not already have a team defined, see link:https://help.github.com/en/github/setting-up-and-managing-organizations-and-teams/creating-a-team[Creating a team, window=_blank].
 
 .To provide authorization:
 * The `Spec.Github.organization` attribute should be set to the organization hosting the collections in the remote GitHub repository.  In the example above, the organization name is `my_org`.
 * The `Spec.Github.teams` attribute should be set to a yaml list of GitHub teams in the remote GitHub repository, whose members can administer the collection using the Kabanero CLI.
 * The `Spec.Github.apiUrl` attribute should be set to the API URL for the remote GitHub repository being used.  For example, the API URL for `github.com` is `https://api.github.com`.
 +
-A modified Kabanero CR instance for a collection repository located in the `my_org` organization of the `github.example.com` GitHub repository using release `0.5.0` of collections might look like this:
+A modified Kabanero CR instance for an application stack repository located in the `my_org` organization of the `github.example.com` GitHub repository using release `0.5.0` of collections might look like this:
```
Comment (stephenkinder) [inline ref/general/configuration/github-authorization.adoc]: This example still shows collections, which is true at the 0.5.0 level; are we expecting that we should update this with a Kabanero CR example?  If so, the CR below and the statement at the end of "line 17" needs to be updated to remove collections and show a 0.6.0+ example.

File: ref/general/configuration/tekton-webhooks.adoc [hunk 3]
```diff
@@ -76,15 +76,15 @@ Select the Docker registry where any built images are pushed. Use the same Docke
 . Update code and merge a PR.
 * Make any update to your code, create a PR, and merge it. This change kicks off a PipelineRun.
 
-. Check the PipelineRun from your Tekton Dashboard.
-* From your Tekton Dashboard, select `PipelineRuns` from the sidebar menu. Look for the PipelineRun you just kicked off.
+. Check the PipelineRun from your pipelinesd dashboard.
```
Comment (stephenkinder) [inline ref/general/configuration/tekton-webhooks.adoc]: "pipelinesd" typo

Comment (jantley-ibm) [reply ref/general/configuration/tekton-webhooks.adoc]: fixed

File: ref/general/configuration/github-authorization.adoc (hunk 2 above)
Comment (jantley-ibm) [reply ref/general/configuration/github-authorization.adoc]: updated example provided by Tim K.

File: ref/general/configuration/console-oauth.adoc (hunk 1 above)
Comment (jantley-ibm) [reply ref/general/configuration/console-oauth.adoc]: simplified language

**Comment by jantley-ibm** [comment]
@stephenkinder updates made, please double-check and approve

File: ref/general/configuration/console-oauth.adoc (hunk 1 above)
Comment (stephenkinder) [reply ref/general/configuration/console-oauth.adoc]: It still contains "communicate with the stack hub server."  I don't know what the stack hub server is.

**Pull request merged**